extensions/amp-story/1.0/_locales/en.js

```diff
@@ -207,6 +207,14 @@ const strings = {
     description:
       'Label for the fourth answer choice from a multiple choice quiz (e.g. D in A/B/C/D)',
   },
+  [LocalizedStringId.AMP_STORY_PAUSE_BUTTON_LABEL]: {
+    string: 'Pause story',
+    description: 'Label for the pause button on the system layer',
```

Comment:
“Without knowing the context of a message, a translator can't do much better than machine translation. Writing good message descriptions will help the translator understand the purpose of the message and its relationship to the surrounding messages.
[...]
Before writing a description, you should realize that translators will not know the product as well as you do. They work on multiple products, and they may not be engineers, so you should write descriptions that are understandable by a general audience.”

Translators won't know what a Story or a system layer is. I'm pinging you a link to good/bad examples.